src/Publisher.ts

```diff
@@ -797,18 +846,21 @@ export default class Publisher {
 			for (let i = 0; i < transclusionMatches.length; i++) {
 				try {
 					const transclusionMatch = transclusionMatches[i];
-					const [tranclusionFileName, headerName] = transclusionMatch
+					const [transclusionFileName, headerName] = transclusionMatch
 						.substring(
 							transclusionMatch.indexOf("[") + 2,
 							transclusionMatch.indexOf("]"),
 						)
 						.split("|");
-					const tranclusionFilePath =
-						getLinkpath(tranclusionFileName);
+					const transclusionFilePath =
+						getLinkpath(transclusionFileName);
 					const linkedFile = this.metadataCache.getFirstLinkpathDest(
-						tranclusionFilePath,
+						transclusionFilePath,
 						filePath,
 					);
+					if (!linkedFile) {
+						continue;
+					}
```

Comment:
also note: added these checks to continue instead of resorting to type-mayhem and error-catching. 

this is the other thing that is "kinda" logic here